app/views/users/new.html.erb

```diff
@@ -0,0 +1,21 @@
+ <%# <form action="/users" method="post" accept-charset="UTF-8"> %>
+  <%# <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>"> %>
+  <%# <label for="username">Username:</label> %>
+  <%# <input type="text" name="user[username]"><br><br> %>
+  <%# <label for="email">Email:</label> %>
+  <%# <input type="email" name="user[email]"><br><br> %>
+  <%# <label for="password">Password:</label> %>
+  <%# <input type="password" name="user[password]"><br><br> %>
+  <%# <input type="submit" value="Submit"> %>
+<%# </form> %>
+
+<%= form_tag("/users", method: "post") do %>
+  <%= label_tag(:username, "Username:") %>
+  <%= text_field_tag(:username) %>
+  <%= label_tag(:email, "Email:") %>
+  <%= text_field_tag(:email) %>
+  <%= label_tag(:password, "Password:") %>
+  <%= password_field_tag(:password) %>
+
+  <%= submit_tag("Save") %>
+<% end %>
```

Comment:
This form implementation should be commented out, as per requirement. 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

Use the link above as a reference.